applications/nrf5340_audio/README.rst

```diff
@@ -898,7 +898,9 @@ For example, to build the application using the script for the application core
 
```

Comment:
Should change from:
"run :file:`buildprog.py` **from** the :file:`applications/nrf5340_audio/tools/buildprog` directory to build the firmware"
to:
"run :file:`buildprog.py` **in** the :file:`applications/nrf5340_audio/tools/buildprog` directory to build the firmware"

And:
"run the following command from the :file:`buildprog` directory:"
to:
"run the following command:"

In the sentences above